Fixed Firefox autoload fails to find .xpi file.

sun opened the pull request:
Removes the version suffix from the build to always build just "dreditor.xpi".  Adding a version suffix is the responsibility of a build/release archiver post-process.

sun committed:
Fixed Firefox autoload fails to find .xpi file.

Removes the version suffix from the build to always build just "dreditor.xpi".  Adding a version suffix is the responsibility of a build/release archiver post-process.

markhalliwell commented on Gruntfile.js:
```diff
@@ -239,8 +239,7 @@ module.exports = function(grunt) {
           "mozilla-addon-sdk": "master",
           extension_dir: "build/firefox",
-          dist_dir: "release",
-          arguments: "--output-file=<%= pkg.name %>-<%= pkg.version %>.xpi"
```
I'd rather not take this out entirely.

markhalliwell commented:
The Chrome and Safari extensions should have the versions taken out too.

sun committed:
Fixed readability/coding-style of build:safari grunt task.

sun committed:
Adjusted all extension builds to use just dreditor.* as artifact filename.

Artifact filenames may clash, so use a vendor-specific subdirectory in ./release for each extension.
